lib/parse.js

```diff
@@ -1,18 +1,42 @@
-export default function parse(fileContent: String): Object {
-  // For some reason, we can't `import` babylon using an es6 import.
+const TYPESCRIPT_FILE_PATH_REGEX = /\.tsx?$/
+
+export default function parse(fileContent: String, absolutePathToFile: String): Object {
+  // For some reason, we can't `import` @babel/parser using an es6 import.
   // eslint-disable-next-line global-require
-  const babylon = require('babylon');
+  const babelParser = require('@babel/parser');
+
+  const typePlugins = TYPESCRIPT_FILE_PATH_REGEX.test(absolutePathToFile)
+    ? ['typescript']
+    : ['flow']
```

Comment:
Not a big deal either way, but why use an array here instead of a string?

Reply:
Just lazy coding 🙈, was unsure how many plugins was required for TS so I coded it as an array before figuring it out.